partiql-logical/src/lib.rs

```diff
@@ -228,6 +228,14 @@ pub enum BindingsOp {
     Sink,
 }
 
+#[derive(Debug, Clone, Default, Eq, PartialEq)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub enum ProjectAllMode {
+    #[default]
+    Unwrap,
+    PassThrough,
```

Comment:
nit: could add some docs for what these variants represent

Reply:
will do in later PR